packages/components/src/internal/app/utils.test.ts

```diff
@@ -639,9 +639,9 @@ describe('utils', () => {
     });
 
     test('isTransformScriptsEnabled', () => {
-        expect(isTransformScriptsEnabled({})).toBe(false);
-        expect(isTransformScriptsEnabled({ inventory: {} })).toBe(false);
-        expect(isTransformScriptsEnabled({ api: { moduleNames: ['premium'] }, })).toBe(true);
+        expect(isTransformScriptsEnabled({})).toBe(true);
+        expect(isTransformScriptsEnabled({ api: { moduleNames: ['premium'] } })).toBe(true);
+
         expect(isTransformScriptsEnabled({ samplemanagement: {} })).toBe(false);
```

Comment:
should add `window.history.pushState({}, 'isApp', '/samplemanager-app.view#');`?

Reply:
Ah yes. Had it, then took it out, then forgot to add it back.